libpod/container_commit.go

```diff
@@ -24,6 +25,8 @@ type ContainerCommitOptions struct {
 	Changes []string
 }
 
+var DefaultVolumeGroups = []string{"cgroup", "devpts", "mqueue", "proc", "shm", "sysfs", "tmpfs"}
```

Comment:
This seems rather flimsy, and likely to break as new Default Volumes get added.  Any way we can generate this list form the spec or from libcontainer?

Reply:
@rhatdan We should be able to get the originals if we add them in `cmd/podman` because we do store the original create CLI